@ecomplus/storefront-template/template/scss/themes/ecom-beauty/_custom.scss

```
@@ -162,4 +194,26 @@ main {
       }
     }
   }
+
+  .blog {
+    .card {
+      border: none;
+    }
+
+    .card-title {
+      a {
+        color: $secondary;
+        font-family: 'Playfair Display', 'Regular Roman';
```

```suggestion
        font-family: 'Playfair Display', 'Regular Roman', sans-serif;
```

É bom sempre ter um fallback no font family por tipo de fonte, assim ele carrega por tipo entre as fontes do sistema caso tenha algum problema carregando a fonte customizada do CDN ou enquanto ela é carregada...